static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -1,35 +1,79 @@
-import {useState} from 'react';
+import {useCallback, useEffect, useState} from 'react';
 import styled from '@emotion/styled';
 
+import {promptsCheck, promptsUpdate} from 'sentry/actionCreators/prompts';
 import {Button} from 'sentry/components/button';
 import Card from 'sentry/components/card';
+import {openConfirmModal} from 'sentry/components/confirm';
+import {DropdownMenu, MenuItemProps} from 'sentry/components/dropdownMenu';
 import ExternalLink from 'sentry/components/links/externalLink';
-import {Tooltip} from 'sentry/components/tooltip';
-import {IconCommit, IconGithub, IconInfo, IconMail} from 'sentry/icons';
+import QuestionTooltip from 'sentry/components/questionTooltip';
+import {IconCommit, IconEllipsis, IconGithub, IconMail} from 'sentry/icons';
 import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import {MissingMember, Organization} from 'sentry/types';
+import {promptIsDismissed} from 'sentry/utils/promptIsDismissed';
+import useApi from 'sentry/utils/useApi';
 import withOrganization from 'sentry/utils/withOrganization';
 
 type Props = {
   missingMembers: {integration: string; users: MissingMember[]};
-  onSendInvite: (email: string) => Promise<void>;
+  onSendInvite: (email: string) => void;
   organization: Organization;
 };
 
 export function InviteBanner({missingMembers, onSendInvite, organization}: Props) {
   // NOTE: this is currently used for Github only
-  // TODO(cathy): include snoozing, docs link
   const [sendingInvite, setSendingInvite] = useState<boolean>(false);
+  const [showBanner, setShowBanner] = useState<boolean>(false);
+
+  const api = useApi();
+  const integrationName = missingMembers?.integration;
+  const promptsFeature = `${integrationName}_missing_members`;
+
+  const snoozePrompt = useCallback(async () => {
+    await promptsUpdate(api, {
+      organizationId: organization.id,
+      feature: promptsFeature,
+      status: 'snoozed',
+    });
+
+    setShowBanner(false);
+  }, [api, organization, promptsFeature]);
+
+  useEffect(() => {
+    promptsCheck(api, {
+      organizationId: organization.id,
+      feature: promptsFeature,
+    }).then(prompt => {
+      setShowBanner(!promptIsDismissed(prompt));
+    });
+  });
 
   if (
+    !organization.features.includes('integrations-gh-invite') ||
+    !showBanner ||
     !organization.access.includes('org:write') ||
     !missingMembers?.users ||
     missingMembers?.users.length === 0
```

Comment:
turn this into a boolean and skip making the request if they aren't going to see anything anyway